dev-docs/modules/userId.md

```diff
@@ -499,6 +524,7 @@ Bidders that want to support the User ID module in Prebid.js, need to update the
 | Unified ID | Trade Desk | bidRequest.userId.tdid | `"2222"` |
 | DigiTrust | IAB | bidRequest.userId.digitrustid | `{data: {id: "DTID", keyv: 4, privacy: {optout: false}, producer: "ABC", version: 2}` |
 | ID5 ID | ID5 | bidRequest.userId.id5id | `"ID5-12345"` |
+| netID | netID | bidRequest.userId.netId | `"fH5A3n2O8_CZZyPoJVD-eabc6ECb7jhxCicsds7qSg"` |
 
 For example, the adapter code might do something like:
 
```

Comment:
Mind describing how the sub adapters need to change. Others have used something like:

`{
                "source": "netId.com",
                "uids": [{
                    "id": "11111111"
                }]
            }`